brainstormforce/ultimate-addons-for-gutenberg
# Fix: Advanced column - Wend Full width cause align default set to wide in next-release.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/2121e94d-d987-4ff5-bc76-6c220614e1bb?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- https://share.getcloudapp.com/WnuynYnB

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop the Advanced column in Master.
- Add a background image and set container width to Theme Container Width.
- Publish and check frontend. Then Switch to our fixed branch.
- duplicate the editor /frontend of the master and Refresh the duplicated editor and fronted and compare it with the master as in the screencast. It should be the same.
- Set container width to custom in master check the same after switching to fixed branch.
- Set alignment and vertical alignment in master and then switch to fixed branch and check editor and frontend. it should be same as master.
- No debug log. 

### Checklist:
